test/e2e/app-dir/ppr-navigations/simple/next.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   experimental: {
     ppr: true,
+    clientSegmentCache: true,
```

Comment:
Just curious, but why are we only enabling it for this PPR test?

Reply:
It's just temporary so the `per-segment-prefetching.test.ts` tests still work. I'm just going to delete that test file in an upcoming PR, since we have plenty of coverage via the e2e tests now, and then I can turn this flag off again. I figured it's fine to turn it on here temporarily because the other tests in this directory are fairly basic smoke tests.